test/performance/graph_classification/pyg/base.py

```diff
@@ -177,16 +184,30 @@ def train(model, train_loader, val_loader, args):
 
     labels = np.array([data.y.item() for data in dataset.test_split])
 
-    train_loader = DataLoader(dataset.train_split, batch_size=args.batch_size)
-    val_loader = DataLoader(dataset.val_split, batch_size=args.batch_size)
-    test_loader = DataLoader(dataset.test_split, batch_size=args.batch_size)
-    
+    def seed_worker(worker_id):
+        #seed =  torch.initial_seed()
+        torch.manual_seed(args.dataset_seed)
+        np.random.seed(args.dataset_seed)
+        random.seed(args.dataset_seed)
+    g = torch.Generator()
+    g.manual_seed(args.dataset_seed)
+
+    train_loader = DataLoader(dataset.train_split, batch_size=args.batch_size, worker_init_fn=seed_worker, generator=g)
+    val_loader = DataLoader(dataset.val_split, batch_size=args.batch_size, worker_init_fn=seed_worker, generator=g)
+    test_loader = DataLoader(dataset.test_split, batch_size=args.batch_size, worker_init_fn=seed_worker, generator=g)
```

Comment:
If the randomness cannot be controlled, we can report average mean, std of 100 trials.